owtf/utils/pycompat.py

```diff
@@ -20,3 +20,10 @@ def u(s):
 
     def u(s):
         return unicode(s)
+
+
+def get_dict_iter_items(dictionary):
```

Comment:
Let's make this more standard as defined in https://www.python.org/dev/peps/pep-0469/#id9 and https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/six/__init__.py#L584.

Reply:
Rename to `get_dict_iteritems()`.